server/api/sessions.js

```diff
@@ -84,6 +113,35 @@ internals.applyRoutes = function (server, next) {
 
     server.route({
         method: 'DELETE',
+        path: '/sessions/my/{id}',
+        config: {
+            auth: {
+                strategy: 'simple',
+                scope: 'account'
+            }
+        },
```

Comment:
I thought we were going to use a pre handler that prevented someone from `DELETE`ing their current session. We should add that.

Note: The conventional way of deleting your current session is to use the `/api/logout` endpoint.